src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/CodeGenerator.cs

```diff
@@ -88,6 +84,14 @@ private static MethodInfo StringFormat
             }
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2070:UnrecognizedReflectionPattern",
+            Justification = "The trimmer will never remove the Invoke method from delegates.")]
+        internal static MethodInfo GetInvokeMethod(Type delegateType)
+        {
```

Comment:
Although out of scope for this change, this should be cacheable as there is a fixed set of delegate types that are passed. The caller should pass the cached Invoke MethodInfo instead of the cached delegate Type